PocketKit/Sources/PocketKit/MyList/SavesContainerViewController.swift

```diff
@@ -128,7 +128,7 @@ class SavesContainerViewController: UIViewController, UISearchBarDelegate {
         }
         navigationItem.searchController?.showsSearchResultsController = true
 
-        searchViewModel.$searchText.sink { searchText in
+        searchViewModel.$searchText.dropFirst().sink { searchText in
```

Comment:
Clarification, please! Is this because `$searchText` is getting published the first time it's subscribed to, which is kind of unnecessary (since it's an empty string)?

Reply:
when 

> @cyndichin specifically for this mention in the PR description:
> 
> > Support pagination for online search results
> 
> Is this simply an update to a constant value? There's nothing code-wise that really adds support for pagination, if I'm reading correctly.

yes, the constant modifies the pagination input, which was already implemented in previous online search work


```
var shouldFetchNextPage = true
var items: [SearchSavedItemParts] = []
var pagination = PaginationInput(first: GraphQLNullable<Int>(integerLiteral: Constants.pageSize))

 while shouldFetchNextPage {
            let filter = getSearchFilter(with: scope)
            let query = SearchSavedItemsQuery(term: term, pagination: .init(pagination), filter: .some(filter))
            let result = try await apollo.fetch(query: query)
            result.data?.user?.searchSavedItems?.edges.forEach { edge in
                items.append(edge.node.savedItem.fragments.searchSavedItemParts)
            }
            if let pageInfo = result.data?.user?.searchSavedItems?.pageInfo {
                pagination.after = pageInfo.endCursor ?? .none
                shouldFetchNextPage = pageInfo.hasNextPage
            } else {
                shouldFetchNextPage = false
            }
            _results = items
        }
```